module-cheatsheets/scRNA-seq-advanced-cheatsheet.md

```diff
@@ -169,6 +169,51 @@ Read the full documentation and download cheatsheets (where available) for these
 
 ### `Seurat` and `SCE` object conversion
 
+This [vignette](https://satijalab.org/seurat/articles/conversion_vignette.html) from the `Seurat` documentation explains how to convert between `SCE` and `Seurat` formats.
+
+In addition, we provide some code examples below for how you can accomplish these conversions.
+For all code examples below, it is assumed that `SingleCellExperiment` and `Seurat` libraries have been loaded in your R environment:
+
+```r
+library(Seurat)
+library(SingleCellExperiment)
+```
+
+
 #### Converting from `Seurat` to `SCE`
 
+The following example code assumes you have a `Seurat` object called `seurat_obj`.
+
+```r
+# Convert Seurat object to SCE object using the
+#  Seurat function `as.SingleCellExperiment()
+sce_object <- as.SingleCellExperiment(seurat_obj)
+```
+
+Alternatively, you can extract individual slots from the `Seurat` object and build your `SCE` object from scratch.
+You can refer to the ["The `SingleCellExperiment` class" chapter in _Orchestrating Single Cell Analyses_](http://bioconductor.org/books/release/OSCA.intro/the-singlecellexperiment-class.html) for approaches to constructing `SCE` objects.
+For example, to obtain an SCE object that contains only the raw counts stored in the `Seurat` object's "RNA" assay:
+
+```r
+# First, extract the counts matrix from the Seurat object
+counts_matrix <- GetAssayData(seurat_obj[["RNA"]])
+
+# Create an SCE object from the counts matrix
+sce_object <- SingleCellExperiment(assays = list(counts = counts_matrix))
+```
+
 #### Converting from `SCE` to `Seurat`
+
+This [documentation from the `ScPCA`](https://scpca.readthedocs.io/en/latest/faq.html#what-if-i-want-to-use-seurat-instead-of-bioconductor) introduces how to convert `SCE` objects to `Seurat` objects.
+Although this documentation was written for `ScPCA` datasets, the steps generally apply to any `SCE object`.
+Briefly, here is how you can convert a `Seurat` to `SCE` object, focusing on porting over _assays_.
```

Comment:
I think we will to show the steps to add at least the cell and feature metadata here. We present it as "optional" in the docs, but we probably shouldn't.

Reply:
Something else I also thought about is whether we should straight-up recommend `scpcaTools::sce_to_seurat()` here, in addition to or instead of individual steps.

Reply:
Hmm, maybe not  ^, since our function only allows _one assay_ to get ported over... But I've stubbed something out already so we'll see.

Reply:
I think you can point people to it as an example, but I would not exactly _recommend_ it.